panels/templates/attractions/attractions_common.html

```diff
@@ -208,6 +212,10 @@
     display: inline-block;
   }
 
+  .hidden-soldout.hidden-modal-form {
+    display: none;
+  }
```

Comment:
I implemented @RobRuana 's suggestion about showing the ``$doneModal`` instead of just a toast and hiding the badge form if we got there from the "I don't know my badge number" form.  I implemented that by adding an ``addClass`` / ``removeClass`` which sets or removes the ``hidden-modal-form`` class appropriately.